python-sdk/nuimages/nuimages.py

```diff
@@ -268,7 +239,7 @@ def list_attributes(self, sort_by: str = 'freq') -> None:
             print(format_str.format(
                 attribute_freqs[attribute['token']], attribute['name'], attribute['description']))
 
-    def list_sensors(self) -> None:
+    def list_cameras(self) -> None:
```

Comment:
Changing the name from `list_sensors` to `list_cameras` might be confusing to the users as `LIDAR_TOP` is listed.

Reply:
LIDAR_TOP is gone now!

Reply:
Ah, I think LIDAR_TOP appeared on my end 'cause I was using the old data files, haha